docs/products/dragonfly/concepts/ha-dragonfly.md

```diff
@@ -50,59 +44,45 @@ access.
-    problems, such as losing an entire node, more drastic recovery
-    measures are required. Aiven's monitoring infrastructure
-    automatically detects a failing node when it reports problems with
-    its self-diagnostics or stops communicating altogether. The
-    monitoring infrastructure then schedules the creation of a new
-    replacement node.
+- **Minor failures**: Aiven automatically handles minor failures, such as service process crashes or temporary loss of network access, without any significant changes to the service deployment. In all plans, the service automatically restores regular operation by restarting crashed processes or restoring network access when available.
+- **Severe failures**: In case of severe hardware or software problems, such as losing an entire node, more drastic recovery measures are required. Aiven's monitoring infrastructure automatically detects a failing node when it reports problems with its self-diagnostics or stops communicating altogether. The monitoring infrastructure then schedules the creation of a new replacement node.
 
 :::note
-In case of database failover, your service's **Service URI** remains
-the same---only the IP address changes to point to the new primary node.
+In case of database failover, your service's
+**Service URI** remains the same---only the IP address changes to
+point to the new primary node.
 :::
 
 ## High availability for business, premium, and custom plans
 
-If a standby Dragonfly node fails, the primary node continues running.
-The system prepares the replacement standby node and synchronizes it
-with the primary for normal operations to resume.
+If a standby Dragonfly node fails, the primary node continues running. The system prepares
+the replacement standby node and synchronizes it with the primary for normal operations
+to resume.
 
-In case the primary Dragonfly node fails, the standby node is evaluated
-for promotion to the new primary based on data from the Aiven monitoring
-infrastructure. Once promoted, this node starts serving clients, and a
-new node is scheduled to become the standby. However, during this
-transition, there may be a brief service interruption.
+In case the primary Dragonfly node fails, the standby node is evaluated for promotion to
+the new primary based on data from the Aiven monitoring infrastructure.
+Once promoted, this node starts serving clients, and a new node is scheduled to become
+the standby. However, during this transition, there may be a brief service interruption.
 
-If the primary and standby nodes fail simultaneously, new nodes will be
-created automatically to replace them. However, this may lead to data
-loss as the primary node is restored from the latest backup. As a
-result, any database writes made since the last backup could be lost.
+If the primary and standby nodes fail simultaneously, new nodes will be created
+automatically to replace them. However, this may lead to data loss as the primary
```

Comment:
```suggestion
If the primary and standby nodes fail simultaneously, new nodes are created
automatically to replace them. However, this may lead to data loss as the primary
```